lib/settings-panel.coffee

```diff
@@ -67,7 +70,7 @@ class SettingsPanel extends View
             appendSetting.call(this, namespace, name, settings[name])
 
   sortSettings: (namespace, settings) ->
-    _.chain(settings).keys().sortBy((name) -> name).sortBy((name) -> atom.config.getSchema("#{namespace}.#{name}")?.order).value()
+    sortSettings(namespace, settings)
```

Comment:
Just curious, why did you move this out into a separate function since it's only used here?